### ISSUE TITLE
[Suggestion] Option to disable fog color change based on light level

### ISSUE THREAD

**Issue by JadeMonsuta** [sp614x/optifine#1741]
For a long time, Minecraft has had this feature where the color of fog will change depending on what lighting level you are in (or depending on if there is a block above you or not)

An option to disable this would be greatly appreciated.

**Comment by RandomMangaFan** [comment]
I agree. In fact, this should be disabled by default if it is added since it causes weird lighting situations when riding minecarts or going on soul sand if the horizon is visible.
